src/core.js

```diff
@@ -464,7 +467,7 @@ function isArrayLike( obj ) {
 	var length = !!obj && "length" in obj && obj.length,
 		type = jQuery.type( obj );
 
-	if ( type === "function" || jQuery.isWindow( obj ) ) {
+	if ( jQuery.isFunction( obj ) || jQuery.isWindow( obj ) ) {
```

Comment:
Doesn't look like it: https://github.com/jquery/jquery/issues?q=is%3Aissue%20isWindow